OneBlink.SDK/models/FormApprovalFlowInstanceStep.cs

```diff
@@ -1,16 +1,8 @@
 using System.Collections.Generic;
 namespace OneBlink.SDK.Model
 {
-    public class FormApprovalFlowInstanceStep
+    public class FormApprovalFlowInstanceStep : FormApprovalFlowStepBase
```

Comment:
`isConditional`, `requiresAllConditionalPredicates`, and `conditionalPredicates` should not be on this class.